### ISSUE TITLE
Illegal filehandle in Lambda

### ISSUE THREAD

**Issue by jssmith** [jssmith/ssqlite#14]
```
read 192.168.1.57/tpcc-nfs-journal offset:0 bytes:1 nfs rpc error illegal filehandle
```

This appears to occur on the second invocation of the Lambda, following one which terminates abnormally.

**Comment by convolvatron** [comment]
can you run with NFS_PACKET_TRACE?
there shouldn't be any state shared between the lambda instances?

**Comment by jssmith** [comment]
I have made some progress on isolating this issue. It appears that the filehandle populated by `file_create` works, whereas that populated by `file_open_read` does not. The proper solution is probably to hook up the latter to  `file_open_internal`.

**Comment by jssmith** [comment]
fixed in `onlambda` branch (81ccea11e916223c03f34e86dd6bc3ae365c6704)
